flags.go

```diff
@@ -174,8 +292,15 @@ func parseFlagsCSV(r io.Reader) (map[string]Flag, time.Time, error) {
 			// TODO also track somehow
 			rate = 0
 		}
-
-		flags[name] = Flag{Name: name, Rate: rate}
+		var active = false
+		var rules = []Rule{}
+		if rate > 0 {
```

Comment:
You could just unconditionally add the RateRule, right? If `Rate: 0`, then it will be equivalent to off